sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/SelectQuery.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using Azure.DigitalTwins.Core.Queries.QueryBuilders;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    // NOTE -- accessibility internal for now to bypass needing static.
+    /// <summary>
+    /// Custom ADT query builder class that facilitates building queries against an ADT instance.
```

Comment:
this is no longer what this class represents, this comment is now more suitable for the AdtQueryBuilder class.